02-microarray/dimension-reduction_microarray_01_pca.Rmd

```diff
@@ -196,14 +196,14 @@ metadata <- readr::read_tsv(metadata_file)
 
 # Read in data TSV file
 df <- readr::read_tsv(data_file) %>%
-  # Tuck away the gene ID  column as rownames
+  # Tuck away the gene ID column as rownames, leaving a numeric matrix
```

Comment:
It's still a data frame though so I don't want them confused. So is there a different way we can say this?